### DISCUSSION TITLE
关于这个项目

### DISCUSSION THREAD

**Discussion by E0x72-21** [hite4044/Minecraft-Server-Ports-Scanner-GUI#16] [Q&A]
这个项目就是这个视频里介绍的扫描器吧？
https://www.bilibili.com/video/BV1aT421X7Y2

**Comment by hite4044** [comment]
对的
你不就是那个Win-Design群里的冒泡制造机吗😅

**Reply by E0x72-21** [reply]
> 对的 你不就是那个Win-Design群里的冒泡制造机吗😅

冒泡怎么你了（恼
